spec/models/github_repository_spec.rb

```diff
@@ -150,6 +150,44 @@
       end
     end
 
+    context "importable" do
+      describe "#importing?", :vcr do
```

Comment:
Right, it doesn't really make sense to test the other methods in the #importing block though. Instead, you can test the importing? method on all 3 cases in the #importing block, and the imported method on all 3 cases in the #Imported block, etc. Then you still get the ensure that the other states aren't true, but it's organized a little better. Does that make sense?